src/flag_gems/runtime/__init__.py

```diff
@@ -0,0 +1,24 @@
+from . import backend, commom_utils, register
+from .config import Config
+from .device import device_ctx
+
+global configer, device
+configer = Config()
+device = device_ctx()
```

Comment:
If this is intended to be a singleton, make it a real singleton.

See also the device in device.py